### PR TITLE
wallet circular balance component + dependencies

### PR DESCRIPTION
**Pull request by jellyfish-tom**
## What does this PR do?
Introduces balance component that is going to be used in Airdrop dashboard.

## Todos:

- [X] Checked on iOS
- [ ] Checked on Android

### CODE REVIEW

File: android/settings.gradle [hunk 1]
```diff
@@ -1,4 +1,6 @@
 rootProject.name = 'GoldWallet'
+include ':react-native-svg'
+project(':react-native-svg').projectDir = new File(rootProject.projectDir, '../node_modules/react-native-svg/android')
```
Comment (marek-siemieniuk-morawski) [inline android/settings.gradle:3]: Finally, support for svg! 🎉

File: src/screens/AirdropDashboard/components/WalletBalanceCircle.tsx [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+
+export const WalletBalanceCircle: FC<Props> = ({ balance = 0, threshold }: Props) => (
+  <AnimatedCircularProgress
+    size={149}
+    width={8}
+    fill={(balance / threshold) * 100}
+    tintColor={palette.textSecondary}
+    backgroundColor={palette.lightGrey}
+  >
+    {() => (
```
Comment (marek-siemieniuk-morawski) [inline src/screens/AirdropDashboard/components/WalletBalanceCircle.tsx:22]: why function here? what does it do?

File: src/screens/AirdropDashboard/components/WalletBalanceCircle.tsx [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+      <View>
+        <Text style={styles.airdropBTCVBalance}>{balance} BTCV</Text>
```
Comment (marek-siemieniuk-morawski) [inline src/screens/AirdropDashboard/components/WalletBalanceCircle.tsx:24]: As far as I remember we have a function for converting a raw amount to the BTCV format. Am I right, @KedziaPawel ?

**Review by marek-siemieniuk-morawski — APPROVED** [review]

File: src/screens/AirdropDashboard/components/WalletBalanceCircle.tsx [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+import { typography, palette } from 'app/styles';
+
+const i18n = require('../../../../loc');
+
+interface Props {
+  balance: number;
+  threshold: number;
+}
+
+export const WalletBalanceCircle: FC<Props> = ({ balance = 0, threshold }: Props) => (
```
Comment (KedziaPawel) [inline src/screens/AirdropDashboard/components/WalletBalanceCircle.tsx:14]: `export const WalletBalanceCircle: FC<Props> = ({ balance = 0, threshold }) => (`
`} :Props` is not needed. 
`balance` is required field so `{ balance, threshold }` should be fine or make balance not required field

File: src/screens/AirdropDashboard/components/WalletBalanceCircle.tsx (hunk 3 above)
Comment (KedziaPawel) [inline src/screens/AirdropDashboard/components/WalletBalanceCircle.tsx:24]: take the unit from consts/models.ts CONST

File: src/screens/AirdropDashboard/components/WalletBalanceCircle.tsx [hunk 5]
```diff
@@ -0,0 +1,43 @@
...
+  <AnimatedCircularProgress
+    size={149}
+    width={8}
```
Comment (KedziaPawel) [inline src/screens/AirdropDashboard/components/WalletBalanceCircle.tsx:17]: lineCap="round" missing

File: src/screens/AirdropDashboard/components/WalletBalanceCircle.tsx [hunk 6]
```diff
@@ -0,0 +1,43 @@
+import React, { FC } from 'react';
```
Comment (KedziaPawel) [inline src/screens/AirdropDashboard/components/WalletBalanceCircle.tsx:1]: it should be a global component as it used on the dashboard and the airdrop details screen after the airdrop

File: src/screens/AirdropDashboard/components/WalletBalanceCircle.tsx (hunk 2 above)
Comment (jellyfish-tom) [reply src/screens/AirdropDashboard/components/WalletBalanceCircle.tsx:22]: you can pass `fill` (percentage value) property back from the CircularComponent, although we don't need it so I omited it. Tried to straight pass React element, but it doesn't work like that, have to be function.

**Pull request merged**